Regular_Falsi/regu.c

```diff
@@ -0,0 +1,49 @@
+// Girlscript Summer Of Code
+// Goldmine problem  in java
```

Comment:
Hey! It would be great if you can make a markdown file and add the explanation in it.